src/routes/notifications/v1/notifications.controller.ts

```diff
@@ -41,10 +134,18 @@ export class NotificationsController {
     @Param('safeAddress', new ValidationPipe(AddressSchema))
     safeAddress: `0x${string}`,
   ): Promise<void> {
-    return this.notificationsService.unregisterSafe({
+    await this.notificationsService.unregisterSafe({
       chainId,
       uuid,
       safeAddress,
     });
+
+    // Compatibility with V2
+    // @TODO Remove NotificationModuleV2 after all clients have migrated and compatibility is no longer needed.
+    await this.notificationServiceV2.deleteSubscription({
+      deviceUuid: uuid as UUID,
```

Comment:
We could use the `UuidSchema` here and not need cast.

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/2063/commits/4e45430b47c033acb313cd5f14162d3b81142b4b